app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -119,7 +137,7 @@ private String getSvmVector(TensorFlow tensorFlow, StudentImage studentImage){
      * Initialize TensorFlow model
      * @return
      */
-    private TensorFlow getInitializedTensorFlow(){
+    private synchronized TensorFlow getInitializedTensorFlow(){
         File modelFile = new File(AiHelper.getModelDirectory(), "vgg_faces.pb");
         if (!modelFile.exists()){
             String logMessage = "Model file: " + modelFile.getAbsolutePath() + " doesn't exist. Please copy it manually";
```

Comment:
@jogrimst Our tutor created the file based on a public file. It's on his dropbox but I copied it to our Google Drive in case he deletes it from there. I will check in the new code in a moment. Then you can see my solution.